modules/metering-uninstall.adoc

```diff
@@ -3,25 +3,63 @@
 // * metering/metering-uninstall.adoc
 
 [id="metering-uninstall_{context}"]
-= Uninstalling metering from {product-title}
+= Uninstalling the Metering Operator from {product-title}
 
-You can remove metering from your cluster.
+You can remove the Metering Operator and the associated metering components from your cluster.
+
+[NOTE]
+====
+Metering does not manage or delete any S3 bucket data. If you used Amazon S3 for storage, any S3 buckets used to store metering data must be manually cleaned up.
+====
+
+== Removing the Metering Operator
+
+You can remove the Metering Operator by following the documentation on xref:../operators/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[deleting Operators from a cluster].
+
+== Uninstalling metering
+
+To uninstall metering, you must remove its custom resource and delete the `openshift-metering` namespace.
 
 .Prerequisites
 
-* Metering must be installed.
+* You have uninstalled Metering following the documentation listed in the previous step.
```

Comment:
Is this prereq referring to deleting the Metering Operator from the cluster? If so, I suggest rewording this:

* The Metering Operator is removed from your cluster.

Reply:
The prerequisite here is that a user has already removed the metering-related OLM resources and not the metering installation itself.